test/parallel/test-os-eol.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const os = require('os');
+
+const eol = common.isWindows ? '\r\n' : '\n';
+
+assert.strictEqual(eol, os.EOL);
+
+common.expectsError(function() {
```

Comment:
I know that we didn't reach consensus on this, but I personally prefer arrow functions in our tests to make them a little more readable.